tests/test_add_start_node.py

```diff
@@ -0,0 +1,38 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver import ActionChains
+from selenium.common.exceptions import NoSuchElementException
+
+web = "https://machinist.flapflap.io/"
+
+
+def test_add_node(driver: WebDriver):
+
+    driver.get(web)
+    source = driver.find_element(By.XPATH, '//*[@id="__next"]/main/div[1]')
+    action = ActionChains(driver)
+    action.double_click(source).perform()
+
+    # check if node is added by verifying if there is text q1
+    inspect_val = driver.find_element(
+        By.XPATH, '//*[@id="__next"]/main/div[1]/div/div[2]/p'
+    ).text
+    true_val = "q1"
```

Comment:
@theNafsters, there is a node when you start up the machinist age due to a bug that is yet to be fixed, and that is the reason why the test passes but once the bug is fixed this test will fail because the first node that is created should be q0 and not q1. The best advice I would give is maybe comment the assertion line till the bug is fixed else this test will fail in future